apis/openapi/slack.com/main/1.7.0/openapi.json

```diff
@@ -23225,27 +23304,30 @@
       }
     },
     "/views.open": {
-      "get": {
-        "parameters": [
-          {
-            "description": "Exchange a trigger to post to the user.",
-            "in": "query",
-            "name": "trigger_id",
-            "required": true,
-            "schema": {
-              "type": "string"
-            }
-          },
-          {
-            "description": "A [view payload](/reference/surfaces/views). This must be a JSON-encoded string.",
-            "in": "query",
-            "name": "view",
-            "required": true,
-            "schema": {
-              "type": "string"
+      "post": {
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
```

Comment:
Should we add `interactivity_pointer` as well here as part of documenting the payload properties?